### PR TITLE
feat(checkout): CHECKOUT-7763 Update Sentry config

### PR DESCRIPTION
**Pull request by bc-peng**
## What?
Update the sentry config so that we can turn on 100% Sentry logging for certain brands.

## Why?
To track all issues for certain stores, so we can get better visibility on issues.

## Depends On (already released)
- https://github.com/bigcommerce/bigcommerce/pull/54892

## Testing / Proof
- CI checks.

### CODE REVIEW

File: packages/core/src/app/checkout/CheckoutApp.tsx [hunk 1]
```diff
@@ -55,23 +40,20 @@ export default class CheckoutApp extends Component<CheckoutAppProps> {
 
     render() {
         return (
-            <ErrorBoundary logger={this.errorLogger}>
```
Comment (animesh1987) [inline packages/core/src/app/checkout/CheckoutApp.tsx:58]: Howcome we are removing the `ErrorBoundary` here?

Comment (animesh1987) [reply packages/core/src/app/checkout/CheckoutApp.tsx:58]: Ignore I see it's moved to the child component.

Comment (bc-peng) [reply packages/core/src/app/checkout/CheckoutApp.tsx:58]: Moved to `checkout.tsx`, that's when we can have a read on the feature flag.
And I don't think we'll compromise on the capture scope, maybe a little.
I was going to comment here, another method is to keep the `ErrorBoundary` here and set it to always 100%. Only use the feature flag to control the error logger in `checkout.tsx`.

File: packages/core/src/app/order/OrderConfirmation.tsx [hunk 2]
```diff
@@ -39,10 +39,11 @@ import { MobileView } from '../ui/responsive';
 import getPaymentInstructions from './getPaymentInstructions';
 import mapToOrderSummarySubtotalsProps from './mapToOrderSummarySubtotalsProps';
 import OrderConfirmationSection from './OrderConfirmationSection';
+import OrderIncompleteHeader from './OrderIncompleteHeader';
 import OrderStatus from './OrderStatus';
 import PrintLink from './PrintLink';
 import ThankYouHeader from './ThankYouHeader';
-import OrderIncompleteHeader from './OrderIncompleteHeader';
+import { BrowserOptions } from '@sentry/browser';
```
Comment (animesh1987) [inline packages/core/src/app/order/OrderConfirmation.tsx:46]: Import order not respected here?

**Review by animesh1987 — APPROVED** [review]

**Pull request merged**